entity-server-api/src/main/java/org/terracotta/entity/ConcurrencyStrategy.java

```diff
@@ -38,20 +41,22 @@
   public final int UNIVERSAL_KEY = Integer.MIN_VALUE; 
   /**
    * MANAGEMENT_KEY is a zero key which is used to manage the entity (anything other than invoke action).
+   * Additionally, a message can choose to use this key if it must be run exclusively within the entity.  Be careful with
+   * the cases where this is done, however, since the MANAGEMENT_KEY cannot be used for synchronizing data.
```

Comment:
"the MANAGEMENT_KEY cannot be used for synchronizing data" - can you explain what you mean by this please?